app/templates/explore.hbs

```diff
@@ -4,6 +4,38 @@
   </div>
   <div class="twelve wide column">
     <h1 class="ui header">{{t 'Events'}}</h1>
+    <div class="ui labels">
+      {{#if filters.category}}
+        <a class="ui mini label">
+          {{filters.category}}
+          <a role="button" {{action 'clearFilter' 'category'}}><i class="icon close"></i></a>
+        </a>
+      {{/if}}
+      {{#if filters.sub_category}}
+        <a class="ui mini label">
+          {{filters.sub_category}}
+          <a role="button" {{action 'clearFilter' 'sub_category'}}><i class="icon close"></i></a>
+        </a>
+      {{/if}}
+      {{#if filters.event_type}}
+        <a class="ui mini label">
+          {{filters.event_type}}
+          <a role="button" {{action 'clearFilter' 'event_type'}}><i class="icon close"></i></a>
+        </a>
+      {{/if}}
+      {{#if filters.start_date}}
+        <a class="ui mini label">
+          {{moment-format filters.start_date 'ddd, MMM DD'}}
+          <a role="button" {{action 'clearFilter' 'start_date'}}><i class="icon close"></i></a>
+        </a>
+      {{/if}}
+      {{#if filters.end_date}}
+        <a class="ui mini label">
+          {{moment-format filters.end_date 'ddd, MMM DD'}}
+          <a role="button" {{action 'clearFilter' 'end_date'}}><i class="icon close"></i></a>
```

Comment:
why wrap it inside an extra anchor tag?

Reply:
I can do tag i with role=button but that will remove the hover effect currently coming due to 'a' tag. Changing i to a leads to css issue of the icon.

Reply:
okay cool, leave it, do make the labels as div instead of a.